grafana-plugin/src/components/IntegrationInputField/IntegrationInputField.tsx

```diff
@@ -47,7 +51,19 @@ export const IntegrationInputField: React.FC<IntegrationInputFieldProps> = ({
   );
 
   function renderInputField() {
-    return <Input className={inputClassName} value={isInputMasked ? value?.replace(/./g, '*') : value} disabled />;
+    let baseClassname = inputClassName;
+    if (isButtonHeight) {
+      baseClassname = 'button-input-height';
+    }
```

Comment:
It seems we make use of it in one case only. This component already supports `inputClassName` prop so how about we just use existing `inputClassName` to apply the height without introducing another prop?